lib/rubocop/cop/bundler/gem_comment.rb

```diff
@@ -58,6 +92,28 @@ def ignored_gem?(node)
           ignored_gems = Array(cop_config['IgnoredGems'])
           ignored_gems.include?(node.first_argument.value)
         end
+
+        def checked_options_present?(node)
+          cop_config[CHECKED_OPTIONS_CONFIG].include?(VERSION_SPECIFIERS_OPTION) && version_specified_gem?(node) \
+          or contains_checked_options?(node)
```

Comment:
```suggestion
          cop_config[CHECKED_OPTIONS_CONFIG].include?(VERSION_SPECIFIERS_OPTION) &&
            version_specified_gem?(node) ||
            contains_checked_options?(node)
```

Reply:
I added parenthesis to make it more explicit, I agree that the use of `or` might be confusing with the different order of precedence.